src/tasks/setup/package.ts

```diff
@@ -67,6 +68,11 @@ export default function taskClean(settings: IGulpSettings): any {
 
     packageInfo.devDependencies = devDependencies;
 
+    packageInfo.scripts = {
+        "gulp": "gulp",
+        "test": "gulp test"
```

Comment:
If your goal is to avoid `npm install -g gulp`, these will need to be the local gulp path. `node_modules/gulp/bin/gulp.js`.